cmd/influx/bucket.go

```diff
@@ -335,6 +343,7 @@ func (b *cmdBucketBuilder) printBuckets(printOpt bucketPrintOpt) error {
 			"ID":              bkt.ID.String(),
 			"Name":            bkt.Name,
 			"Retention":       bkt.RetentionPeriod,
+			"Shard duration":  bkt.ShardGroupDuration,
```

Comment:
Need to think through what this will look like when listing objects returned by InfluxDB Cloud.